sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -148,18 +140,18 @@ namespace sycl::ext::oneapi::experimental::matrix {
   template <typename Group, typename T, size_t NumRows, size_t NumCols,
```

Comment:
Ideally I think these functions should be exposed in the simplest way possible that provides all the necessary information to the user for their proper use. There is some information that I think can be omitted, such as the Group template parameter for example because it is inferred from the Group argument and we never want the user to type it out.

It would probably be a good idea to have a short description for the purpose of the `Space` template parameter that references shared memory, because utilizing shared memory will be very important for application users of `joint_matrix`